src/modules/Unity/Application/mirsurface.cpp

```diff
@@ -530,6 +530,15 @@ void MirSurface::close()
     }
 }
 
+void MirSurface::forceClose()
+{
+    INFO_MSG << "()";
```

Comment:
What does this do?

Reply:
It prints a debug message to know where it is.

Forexample:
```
[2019-06-29:22:41:58.713] qtmir.surfaces: MirSurface[0x29db398,"openstore.openstore-team_openstore"]::forceClose()
```

Reply:
Funny but ok then ^^ - I guess it would be 2 empty brackets